spec/features/reservations_spec.rb

```diff
@@ -552,7 +552,6 @@
       select('10', from: 'items_per_page') # show more than 1 item (default)
```

Comment:
I'm not sure, I think it might fall under unnecessary defaults because we now only create multiple items when necessary (I actually don't think that it's needed here because there is only one model/item in over half of its nested shared examples; I think we should move it to the ones where it's necessary)